src/System.Security.Permissions/ref/System.Security.Permissions.cs

```diff
@@ -70,17 +90,22 @@ public partial class HostProtectionException : System.SystemException {
     public System.Security.Permissions.HostProtectionResource ProtectedResources { get { throw null; } }
     public override string ToString() => base.ToString();
   }
-  public partial class HostSecurityManager {
-    public HostSecurityManager() { }
-    public virtual System.Security.Policy.PolicyLevel DomainPolicy { get { throw null; } }
-    public virtual System.Security.HostSecurityManagerOptions Flags { get { throw null; } }
-    public virtual System.Security.Policy.ApplicationTrust DetermineApplicationTrust(System.Security.Policy.Evidence applicationEvidence, System.Security.Policy.Evidence activatorEvidence, System.Security.Policy.TrustManagerContext context) { throw null; }
-    public virtual System.Security.Policy.Evidence ProvideAppDomainEvidence(System.Security.Policy.Evidence inputEvidence) { throw null; }
-    public virtual System.Security.Policy.Evidence ProvideAssemblyEvidence(System.Reflection.Assembly loadedAssembly, System.Security.Policy.Evidence inputEvidence) { throw null; }
-    [System.ObsoleteAttribute]
-    public virtual System.Security.PermissionSet ResolvePolicy(System.Security.Policy.Evidence evidence) { throw null; }
-  }
-  [System.FlagsAttribute]
+    public partial class HostSecurityManager
```

Comment:
Is it possible to leave the formatting as before? Seems like types were 2 spaces indented. 
I think there is a tool we sometimes run that generates them this way and then it will all show up as a diff.